develop/2017-04-29-fabiocorreacordeiro-Local-Transportation.py

```diff
@@ -0,0 +1,568 @@
+
+# coding: utf-8
+
+# # Exploratory Descriptive Analysis of Local Transportation
+# 
+# This is an Exploratory Descriptive Analysis of expenditure with local transportation. We already created three separated notebooks to explore the data of the subquotas 'Taxi, toll and parking', 'Automotive vehicle renting or charter' and 'Fuels and lubricants'. In this notebook, we will join the analysis of this subquotas.
+# 
+# The subquota 'Automotive vehicle renting or charter' changed in the time. It started like 'Automotive vehicle renting or watercraft charter' and in October/2013 was split in 'Watercraft renting or charter' and 'Automotive vehicle renting or charter'. 
+# 
+# The previous notebooks are in:
+# 
+# 'Automotive vehicle renting or charter': https://github.com/datasciencebr/serenata-de-amor/pull/206/commits/6afda4344c6565ec01536242353730b14f2ebaee

```

Comment:
When I started these analyses I did three notebooks, but the main conclusion is in this last one. I think it isn't really necessary to link all them.